main.py

```diff
@@ -67,7 +60,7 @@ def main(config):
             float_precision=config.float_precision,
             maximize=maximize,
         )
-        surrogate.fit(train_data)
+        surrogate.fit_surrogate(train_data)
```

Comment:
Why not just `fit()`? Isn't `surrogate.fit_surrogate()` redundant?

Reply:
maybe the naming is still a bit off. I thought that there might be special cases where we also want to fit an acquisition function. then we could have fit_surrogate and fit_acquisition or sth

but I can change it to fit

Reply:
I refactored the acquisition part